client/angular/components/auth/signup_form/auth_signup_form.coffee

```diff
@@ -3,27 +3,39 @@ EventBus    = require 'shared/services/event_bus'
 AuthService = require 'shared/services/auth_service'
 I18n        = require 'shared/services/i18n'
 
+{ hardReload }    = require 'shared/helpers/window'
 { submitOnEnter } = require 'shared/helpers/keyboard'
 
+validSignup = (vars, user) ->
```

Comment:
plz move to AuthService, I'm gonna be a real stickler about adding stuff to controllers from now on :)